src/Oci8/Query/OracleBuilder.php

```diff
@@ -247,4 +248,68 @@ public function clone()
     {
         return clone $this;
     }
+
+    /**
+     * Add a basic where clause to the query.
+     *
+     * @param  \Closure|string|array  $column
+     * @param  mixed  $operator
+     * @param  mixed  $value
+     * @param  string  $boolean
+     * @return \Illuminate\Database\Query\Builder|\Yajra\Oci8\Query\OracleBuilder
+     */
+    public function where($column, $operator = null, $value = null, $boolean = 'and')
+    {
+        if ($this->isJsonColumn($column)) {
+            [$column, $jsonKeys] = $this->getJsonColumnAndKeys($column);
+
+            return parent::where(parent::raw("json_value($column, '$.$jsonKeys')"), $operator, $value, $boolean);
+        }
+
+        return parent::where($column, $operator, $value, $boolean);
+    }
+
+    /**
+     * Add a "where not in" clause to the query.
+     *
+     * @param  string  $column
+     * @param  mixed  $values
+     * @param  string  $boolean
+     * @return \Illuminate\Database\Query\Builder|\Yajra\Oci8\Query\OracleBuilder
+     */
+    public function whereNotIn($column, $values, $boolean = 'and')
+    {
+        if ($this->isJsonColumn($column)) {
+            [$column, $jsonKeys] = $this->getJsonColumnAndKeys($column);
+
+            return parent::whereNotIn(parent::raw("json_value($column, '$.$jsonKeys')"), $values, $boolean);
+        }
+
+        return parent::whereNotIn($column, $values, $boolean);
+    }
+
+    /**
+     * Get column and json keys.
+     *
+     * @param  string  $column
+     * @return array
+     */
+    private function getJsonColumnAndKeys(string $column): array
+    {
+        $keys = explode('->', $column);
+        $jsonKeys = implode('.', array_slice($keys, 1));
+
+        return [$keys[0], $jsonKeys];
```

Comment:
I suggest we wrap the column here:

```php
return [$this->grammar->wrap($keys[0]), $jsonKeys];
```

Reply:
I used OraclrGrammer for this and delete override methods from builder.
Everything works correctly